pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -6014,4 +6447,71 @@ def solve_model(model, options=None):
         )
 
     results.write()
+
+    if model.config.hydraulics != Hydraulics.false:
+        mh = pipeline_hydraulics(model, opt)
+
+        if model.config.hydraulics != Hydraulics.post_process:
```

Comment:
```suggestion
        if model.config.hydraulics == Hydraulics.post_process:
```
As discussed on Slack.

Reply:
done